tools/apiview/parsers/APIViewTreeStyleParserSchema/src/decorators.ts

```diff
@@ -0,0 +1,33 @@
+import { DecoratorContext, Operation, Program } from "@typespec/compiler";
+import { StateKeys, reportDiagnostic } from "./lib.js";
+
+export const namespace = "ApiViewTreeStyleParserSchema";
+
+/**
+ * __Example implementation of the `@alternateName` decorator.__
+ *
+ * @param context Decorator context.
+ * @param target Decorator target. Must be an operation.
+ * @param name Alternate name.
+ */
+export function $alternateName(context: DecoratorContext, target: Operation, name: string) {
```

Comment:
Why are you creating a new decorator? APIView isn't an emitter or library. The TypeSpec is just the definition.